src/pages/policies/[policyId].svelte

```diff
@@ -49,8 +41,8 @@ $: members = policy.members || []
 
 $: policyId && loadItems(policyId)
 // sort items so inactive is last
-$: items = $selectedPolicyItems.filter(showApprovedOnly ? itemIsApproved : itemIsNotInactive)
-$: itemsForTable = showAllItems ? $selectedPolicyItems : items.slice(0, 15)
+$: items = $selectedPolicyItems.slice(0, 15)
+$: itemsForTable = hideInactive ? items.filter(itemIsApproved) : items
```

Comment:
Do you want to just use the `approvedItems` variable from the next line (probably pull it above this one, if so)?
```suggestion
$: itemsForTable = hideInactive ? approvedItems : items
```